sql/expression/function/json/json_remove_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package json
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/src-d/go-errors.v1"
+
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/types"
+)
+
+func TestRemove(t *testing.T) {
+	_, err := NewJSONRemove()
+	require.True(t, errors.Is(err, sql.ErrInvalidArgumentNumber))
+
+	f1 := buildGetFieldExpressions(t, NewJSONRemove, 2)
+
+	f2 := buildGetFieldExpressions(t, NewJSONRemove, 3)
+
+	json := `{"a": 1, "b": [2, 3], "c": {"d": "foo"}}`
+
+	testCases := []struct {
+		f        sql.Expression
+		row      sql.Row
+		expected interface{}
+		err      error
+	}{
+		{f1, sql.Row{json, "$.a"}, `{"b": [2, 3], "c": {"d": "foo"}}`, nil},                                            // update existing
```

Comment:
(minor) some of the test case descriptions are a little off, not a big deal, still pretty easy to understand the tests, but would be nice to take a quick pass
```suggestion
		{f1, sql.Row{json, "$.a"}, `{"b": [2, 3], "c": {"d": "foo"}}`, nil},                                            // remove existing
```